api-acceptance-tests/src/test/resources/cucumber/AliasExample.feature

```diff
@@ -0,0 +1,11 @@
+@ui-demi-alias
+Feature: Demonstrate use of aliases to keep tests readable with complex data
+
+  Scenario: Send a message between two users
+    Given the following users:
+      | alias |
```

Comment:
For List in cucumber, you don't need to state a header, in this case testWorld will have an extra user called "alias"